schedule/virt_autotest/install_guest_on_slem_kvm_host.yaml

```diff
@@ -0,0 +1,51 @@
+---
+name: install_guest_on_slem_kvm_host.yaml
+description: |
+  Maintainer: Wayne Chen (wchen@suse.com)
```

Comment:
Would you please add qe-virt@suse.de as secondary maintainer for the new files? This is recently discussed in qe-sync meeting that adding group ML has benefit.

Reply:
Done.